dcf/management/commands/notification_manager.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding:utf-8 -*-
+
+from datetime import datetime as dt, timedelta
+from django.utils.translation import ugettext as _
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.core.mail import send_mail
+
+
+from dcf.models import Item
+
+DAY = 1
+INT_DAYS_BEFORE_START_NOTIFICATION = 2 * DAY
+
+timedelta_start_notification_period = timedelta(
+    days=INT_DAYS_BEFORE_START_NOTIFICATION-(
+        settings.DCF_ITEM_MAX_AGE_DAYS or 0))
+
+timedelta_end_notification_period = timedelta_start_notification_period + (
+    timedelta(days=DAY))
+
+notification_period = [
+    dt.now() + timedelta_start_notification_period,
+    dt.now() + timedelta_end_notification_period]
+
+NON_ACTIVE_DAY = dt.now() - timedelta(
+    days=-settings.DCF_ITEM_MAX_AGE_DAYS or 0)
+
+
+class Command(BaseCommand):
+    """
+    Command for status change and notification users
+    """
+    help = 'Check status per day '
+
+    def handle(self, *args, **options):
+        """ Sending messages """
+        if settings.DCF_ITEM_MAX_AGE_DAYS is None:
+            return
+        items = Item.objects.select_related('user')
+        for item in items.filter(
+                posted__range=notification_period,
```

Comment:
If user have more that one outdated item you will send multiple emails, should be only one with all outdated items for that user.